lib/chutney/TorNet.py

```diff
@@ -761,7 +761,7 @@ def _genRouterKey(self):
         stdouterr = run_tor(cmdline)
         fingerprint = "".join((stdouterr.rstrip().split('\n')[-1]).split()[1:])
         if not re.match(r'^[A-F0-9]{40}$', fingerprint):
-            print("Error when getting fingerprint using '%r'. It output '%r'."
+            print("Error when getting fingerprint using '{0}'. It output '{1}'."
```

Comment:
Please don't make this change, we want the raw data here.

See:
https://stackoverflow.com/questions/6005159/when-to-use-r-instead-of-s-in-python#6005180

Reply:
@teor2345 
pylint says too many arguments for format string with `%r`.

Can't we use this instead ->
`            print("Error when getting fingerprint using '{0}'. It output '{1}'."`
`                  .format(repr(" ".join(cmdline)), repr(stdouterr)))`

Reply:
That's because str.format(…) uses "{}", and print(_, …) uses "%r".

Yes, we can call repr() explicitly.